src/file/davposix.cpp

```diff
@@ -541,6 +664,7 @@ dav_ssize_t DavPosix::preadVec(DAVIX_FD* fd, const DavIOVecInput * input_vec,
 }
 
 ssize_t DavPosix::write(DAVIX_FD* fd, const void* buf, size_t count, Davix::DavixError** err){
+    static bool isProxy = getenv("XRDCL_PROXY") != 0;
```

Comment:
This line is the one which enables the new "multipart upload". Can you place a comment here with context on what enabling `isProxy` does?

Reply:
Will do.